ecosystem/sep-0031.md

```diff
@@ -504,12 +510,12 @@ Name | Type | Description
+`amount_in` | string | (optional) The amount of the Stellar asset received or to be received by the Receiving Anchor. It should be equals to the `amount` value used in the [`POST /transactions`](#post-transactions) request. If a `quote_id` was provided in the [`POST /transactions`](#post-transactions) request, the quote's `sell_amount` should also be equals to `amount_in`. </br></br>The Receiving Anchor may extract fees from this amount before converting it to the amount_out.
 `amount_in_asset` | string | (optional) The asset received or to be received by the Receiving Anchor. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_out` | string | (optional) The amount sent by the Receiving Anchor to Receiving Client.
-`amount_out_asset` | string | (optional) The asset delivered to the Receiving Client. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
-`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_out` | string | (optional) The amount sent or to be sent by the Receiving Anchor to the Receiving Client. When using a `quote_id`, `amount_out` should always be the same as the quote's `buy_amount`. When using a `destination_asset` in the [`POST /transactions`](#post-transactions) request, it's expected that this value is only populated after the Receiving Anchor receives the incoming payment.
+`amount_out_asset` | string | (optional) The asset delivered to the Receiving Client. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. If `destination_asset` was used when creating the transaction, `amount_out_asset` should be equals to `destination_asset`. </br></br>The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor. If a `quote_id` was used, fees should be detailed in the quote's `price_details` field.
+`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if `destination_asset` was included in the `POST /transactions` request and the value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). If a `quote_id` was used, fees and fee assets should be detailed in the quote's `price_details` field.
```

Comment:
If a quote_id was used, `amount_fee_asset` field should be left empty and detailed in the quote's price_details field.

Reply:
https://github.com/stellar/stellar-protocol/pull/1199/commits/5b12d88086c6a0e6579c442a34b752e77ef4e222